src/record/delete/validator/index.ts

```diff
@@ -0,0 +1,57 @@
+import type { KintoneRestAPIClient } from "@kintone/rest-api-client";
+import type { RecordNumber } from "../types/field";
+import {
+  isValidRecordNumber,
+  hasAppCode,
+  getAllRecordIds,
+  getRecordIdFromRecordNumber,
+} from "../usecases/delete/record";
```

Comment:
@mshrtsr 

I considered the place of this folder. But, not sure which one is the best.

1. As current, record/delete/validator
2. record/delete/usecases/validator
3. record/delete/usecases/delete/validator

For `#2`, `validator` is not a use case, like `delete` or `deleteAll`, so It's better to go with `#3`.
However, if we choose `#3`, the folder is too deep. e.g.
src/record/delete/usecases/delete/validator/__test__/fixtures

For `#1`, I thought It can be the same as `parser`, just a tool that serves the delete command. And It can be reused for other use-cases of the delete command later, such as delete by query, ID, ...

What do you think?